moto/events/responses.py

```diff
@@ -42,6 +42,20 @@ def request_params(self):
     def _get_param(self, param, if_none=None):
         return self.request_params.get(param, if_none)
 
+    def _create_response(self, result):
+        """
+        Creates a proper response for the API.
+
+        It basically transforms a dict-like result from the backend
+        into a tuple (str, dict) properly formatted.
+        Args:
+            result (dict): result from backend
+
+        Returns:
+            (str, dict): dumped result and headers
+        """
+        return json.dumps(result), self.headers
```

Comment:
nit: Not sure if we need to return `self.headers` here - looks like those are the request headers, right? I think we could remove the second parameter, and use only `return json.dumps(result)`. 

(But I agree that it makes sense to encapsulate the logic in `_create_response(..)` here, as we may want to add additional logic in the future, potentially.)

Reply:
Damn! You're right, it should be self.response_headers! Next PR thats change. 

Also, I can remove it. I followed the pattern already in place which returns the headers along the body for this [line](https://github.com/localstack/moto/blob/e61cbbeadd6bad9554f07641895e058702a3688b/moto/core/responses.py#L426) to parse.